.github/workflows/lint.yml

```diff
@@ -12,18 +12,6 @@ jobs:
       - name: install node dependencies
         if: always()
         run: make node_modules
-      - name: Validate CSS
-        if: always()
-        run: node_modules/css-validator/bin/css-validator project/frontend/src/*.css
-      - name: Validate JavaScript
-        if: always()
-        run: node_modules/eslint/bin/eslint.js project/frontend/src/*.js
-      - name: Check CSS Formatting
-        if: always()
-        run: node_modules/prettier/bin-prettier.js -c project/frontend/src/*.css
-      - name: Check JavaScript Formatting
-        if: always()
-        run: diff -u <(cat project/frontend/src/*.js) <(node_modules/clang-format/bin/linux_x64/clang-format --style=Google project/frontend/src/*.js)
```

Comment:
Can you comment this out instead, and leave a TODO or an explanation of why it is commented out.

I guess you removed it from a view places, so a TODO everywhere may become messing. Leaving it up to you

Reply:
@leonnemets Will add a comment in the merge of this PR explaining why it was removed. Would that be fine?